src/map/battle.cpp

```diff
@@ -3228,8 +3228,11 @@ struct Damage battle_calc_skill_base_damage(struct Damage wd, struct block_list
 					(!skill_id && sc && sc->data[SC_CHANGE]?4:0);
 
 				wd.damage = battle_calc_base_damage(src, sstatus, &sstatus->rhw, sc, tstatus->size, i);
-				if (is_attack_left_handed(src, skill_id))
+				wd.damage = battle_attr_fix(src, target, wd.damage, ELE_NEUTRAL, tstatus->def_ele, tstatus->ele_lv);
+				if (is_attack_left_handed(src, skill_id)) {
 					wd.damage2 = battle_calc_base_damage(src, sstatus, &sstatus->lhw, sc, tstatus->size, i);
+					wd.damage2 = battle_attr_fix(src, target, wd.damage2, ELE_NEUTRAL, tstatus->def_ele, tstatus->ele_lv);
+				}
```

Comment:
how about this?
wd.damage = battle_calc_base_damage(src, sstatus, &sstatus->rhw, sc, tstatus->size, i);
					if (skill_id == MO_EXTREMITYFIST) {
						wd.damage = battle_attr_fix(src, target, wd.damage, ELE_NEUTRAL, tstatus->def_ele, tstatus->ele_lv);
					}	
				if (is_attack_left_handed(src, skill_id))
					wd.damage2 = battle_calc_base_damage(src, sstatus, &sstatus->lhw, sc, tstatus->size, i);
					if (skill_id == MO_EXTREMITYFIST) {
						wd.damage2 = battle_attr_fix(src, target, wd.damage2, ELE_NEUTRAL, tstatus->def_ele, tstatus->ele_lv);
					}

Reply:
Will work perfectly as you want, but probably isn't correct. Maybe other skills are influenced by this way? I don't know if it's all, so, the perfect one will be a switch-statement to all possible skills affected by this.